ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/converters/RequestConverter.java

```diff
@@ -259,4 +258,34 @@ public static String getResponseType(final ImmutableList<Ds3ResponseCode> respon
                 new Parameter(ParameterModifier.CONST, "char", "payload", ParameterPointerType.SINGLE_POINTER, true)); // String
         return requestPayloadMap.build();
     }
+
+    /**
+     * Retrieves the documentation for the specified Request.
+     * @param requestName The normalized SDK request name with no path
+     */
+    private static String toRequestDocs(
+            final String requestName,
+            final Ds3DocSpec docSpec) {
+        final Optional<String> documentation = docSpec.getRequestDocumentation(requestName);
+        if (!documentation.isPresent()) {
```

Comment:
This can be replaced with:

```java
final Optional<String> documentation = ...
return documentation.orElseGet(() -> {
    LOG.debug("Cannot generate java documentation for request because there is no documentation descriptor: {}", requestName);
    return "";
});
```